certbot/compat/misc.py

```diff
@@ -2,42 +2,31 @@
 This compat module handles various platform specific calls that do not fall into one
 particular category.
 """
-import ctypes
-import errno
+from __future__ import absolute_import
+
+import os  # pylint: disable=os-module-forbidden
```

Comment:
Can we continue to use `certbot.compat.os` here? If not, why not?

Reply:
It is temporary, until a future PR in the part3 plan of windows file permissions makes the complete modifications for `misc`.

Reply:
Then can we use `certbot.compat.os` for now in this PR and change it later when it's necessary?

Reply:
Done.